src/serializers/graph_extensions.c

```diff
@@ -74,6 +75,12 @@ void Serializer_Graph_SetNodeLabels
 	RG_Matrix node_labels   = Graph_GetNodeLabelMatrix(g);
 	GrB_Matrix node_labels_m = RG_MATRIX_M(node_labels);
 
+#if RG_DEBUG
+	GrB_Index nvals;
+	GrB_Matrix_nvals(&nvals, node_labels_m);
+	ASSERT(nvals == 0);
+#endif
```

Comment:
@AviAvni 
this is the right check to perform but located at the wrong place, it should be enforced by the caller,
can you please move it to: src/serializers/decoders/current/v11/decode_graph.c
right before calling `Serializer_Graph_SetNodeLabels(g);` ?